src/main/java/TransactionDao.java

```diff
@@ -0,0 +1,98 @@
+import java.sql.*;
+
+public class TransactionDao {
+
+    private Connection connection = connect();
+    private PreparedStatement preparedStatement = null;
```

Comment:
To może byc chyba zmienną lokalną

Reply:
Niby może ale po co kilka razy pisać to samo ;p

Reply:
Ponieważ dobrą praktyką jest zawężanie widoczności jakiejś zmiennej tylko do takiego zakresu, w jakim jest potrzebna - tak więc jeśli coś może być zmienną lokalną, to niech nią będzie :P Dodatkowo zmienne lokalne są bezpieczniejsze z perspektywy wielowątkowości